File: 4th Sem/Java/Programs/Week 1-practice/Electricity.java
class Electricity {
	public static void main(String[] args) {
		float units = 250;
		double billPay = 0;

		if(units < 100) {
			billPay = units * 1.20;
		}
		else if (units >= 100 && units < 199) {
			billPay = (100 * 1.20) + (units - 100) * 2.10;
		}
		else if (units >= 200 && units < 299) {
			billPay = (100 * 1.20) + (200 * 2.10) + (units - 300) * 3.50;
		}
		else { 
			billPay = (100 * 1.20) + (200 * 2.10) + (300 * 3.50) + (units - 400) * 4.20;
		}
		System.out.println("Bill to pay : " + billPay);
	}
}

File: 2nd Sem/C++/Programs/electricity.cpp
#include<iostream> 
using namespace std;
class ebill 
{ 
     private: 
     int cno; 
     char cname[20]; 
     int units; 
     double bill; 
     public: 
                void get() 
        { 
                 cout<<"Enter Customer No : "<<endl; 
                 cin>>cno; 
                 cout<<"\nEnter Customer Name : "<<endl; 
                 cin>>cname; 
                 cout<<"\nEnter No. of Units used : "<<endl; 
                 cin>>units; 
         } 
                void put() 
         { 
                cout<<"\nCustomer No is : "<<cno; 
                cout<<"\nCustomer Name is : "<<cname; 
                cout<<"\nNumber of Units Consumed : "<<units; 
                cout<<"\nBill of Customer : "<<bill; 
         } 
                void calc() 
        { 
                if(units<=100) 
                       bill=units*1.20; 
               else if(units<=300) 
                       bill=100*1.20+(units-100)*2; 
               else 
                       bill=100*1.20+200*2+(units-300)*3; 
         } 
}; 
             int main() 
     { 
             ebill p1; 
             p1.get(); 
             p1.calc(); 
             p1.put();  
             return 0;
      } 